Neos.Flow/Classes/Core/LockManager.php

```diff
@@ -79,8 +79,12 @@ protected function removeExpiredLock()
         if (filemtime($this->lockFlagPathAndFilename) >= (time() - self::LOCKFILE_MAXIMUM_AGE)) {
             return;
         }
-        @unlink($this->lockFlagPathAndFilename);
-        @unlink($this->lockPathAndFilename);
+        try {
+            @unlink($this->lockFlagPathAndFilename);
+            @unlink($this->lockPathAndFilename);
+        } catch (\Throwable $e) {
+            // PHP 8 apparently throws for unlink even with shutup operator, but we really don't care at this place. It's also the only way to handle this race-condition free.
+        }
```

Comment:
```suggestion
        try {
            @unlink($this->lockFlagPathAndFilename);
        } catch (\Throwable $e) {
            // PHP 8 apparently throws for unlink even with shutup operator, but we really don't care at this place. It's also the only way to handle this race-condition free.
        }
        try {
            @unlink($this->lockPathAndFilename);
        } catch (\Throwable $e) {
        }
```

Just noticed we need two trys because otherwise the second file might not get deleted.